src/codegen/codegen_coreneuron_cpp_visitor.cpp

```diff
@@ -1426,11 +1426,7 @@ void CodegenCoreneuronCppVisitor::print_mechanism_register() {
                           net_recv_init_arg);
     }
     if (info.for_netcon_used) {
-        // index where information about FOR_NETCON is stored in the integer array
-        const auto index =
-            std::find_if(info.semantics.begin(), info.semantics.end(), [](const IndexSemantics& a) {
-                return a.name == naming::FOR_NETCON_SEMANTIC;
-            })->index;
+        const auto index = position_of_int_var(naming::FOR_NETCON_VARIABLE);
```

Comment:
Probably missing something obvious here, but why was `FOR_NETCON_SEMANTIC` replaced with `FOR_NETCON_VARIABLE` here?

Reply:
The one searches `info.semantics` and the other searches `info.integer_variables`.